### PR TITLE
feat(PageHeader): adds separator prop for border-bottom

### PR DESCRIPTION
**Pull request by stutirao**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

**Comment by aditya-kumawat** [comment]
Please update commit message and PR title as per convention.

File: core/components/organisms/pageHeader/PageHeader.tsx [hunk 1]
```diff
@@ -42,14 +42,17 @@ export interface PageHeaderProps extends BaseProps {
    * Page header layout type
    */
   navigationPosition?: navigationPositionType;
+  seperator?: boolean;
 }
 
 export const PageHeader = (props: PageHeaderProps) => {
-  const { title, navigation, actions, tabs, breadcrumbs, badge, status, meta, navigationPosition, className } = props;
+  const { title, navigation, actions, tabs, breadcrumbs, badge, seperator, status,
+    meta, navigationPosition, className } = props;
```
Comment (aditya-kumawat) [inline core/components/organisms/pageHeader/PageHeader.tsx]: Separate everyone into a new line.

File: core/components/organisms/pageHeader/PageHeader.tsx [hunk 2]
```diff
@@ -42,14 +42,17 @@ export interface PageHeaderProps extends BaseProps {
...
   const baseProps = extractBaseProps(props);
 
   const wrapperClasses = classNames({
     'PageHeader-wrapper': true,
+    ['PageHeader-borderBottom']: seperator,
```
Comment (aditya-kumawat) [inline core/components/organisms/pageHeader/PageHeader.tsx]: This should be a modifier.

File: css/src/components/pageHeader.css [hunk 3]
```diff
@@ -1,7 +1,9 @@
 .PageHeader-wrapper {
-  border-bottom: var(--border);
   padding: var(--spacing) var(--spacing-2);
 }
+.PageHeader-borderBottom{
```
Comment (aditya-kumawat) [inline css/src/components/pageHeader.css]: .PageHeader--borderBottom

File: core/components/organisms/pageHeader/PageHeader.tsx [hunk 4]
```diff
@@ -42,14 +42,26 @@ export interface PageHeaderProps extends BaseProps {
...
+    navigation,
+    actions,
+    tabs,
+    breadcrumbs,
+    badge,
+    seperator,
+    status,
+    meta,
+    navigationPosition,
+    className } = props;
```
Comment (aditya-kumawat) [inline core/components/organisms/pageHeader/PageHeader.tsx]: title and className in new lines too

**Comment by aditya-kumawat** [comment]
Update PR/commit message as:
feat(PageHeader): adds separator prop for border-bottom

File: core/components/organisms/pageHeader/PageHeader.tsx [hunk 5]
```diff
@@ -42,14 +42,28 @@ export interface PageHeaderProps extends BaseProps {
...
+    status,
+    meta,
+    navigationPosition,
+    className
+  } = props;
   const baseProps = extractBaseProps(props);
 
   const wrapperClasses = classNames({
     'PageHeader-wrapper': true,
+    ['PageHeader--borderBottom']: seperator,
```
Comment (aditya-kumawat) [inline core/components/organisms/pageHeader/PageHeader.tsx]: PageHeader--borderBottom => PageHeader-wrapper--borderBottom

File: core/components/organisms/pageHeader/PageHeader.tsx [hunk 6]
```diff
@@ -42,14 +42,28 @@ export interface PageHeaderProps extends BaseProps {
    * Page header layout type
    */
   navigationPosition?: navigationPositionType;
+  seperator?: boolean;
```
Comment (aditya-kumawat) [inline core/components/organisms/pageHeader/PageHeader.tsx:49]: Add description.
Add default value as `true`

File: core/components/organisms/pageHeader/PageHeader.tsx [hunk 7]
```diff
@@ -42,14 +42,28 @@ export interface PageHeaderProps extends BaseProps {
...
+    status,
+    meta,
+    navigationPosition,
+    className
+  } = props;
   const baseProps = extractBaseProps(props);
 
   const wrapperClasses = classNames({
     'PageHeader-wrapper': true,
+    ['PageHeader-wrapper--borderBottom']: seperator,
```
Comment (aditya-kumawat) [inline core/components/organisms/pageHeader/PageHeader.tsx]: Change class name to 'PageHeader-wrapper--separator'.

**Review by aditya-kumawat — APPROVED** [review]

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**